src/Symfony/Component/DependencyInjection/Loader/Configurator/Traits/BindTrait.php

```diff
@@ -16,6 +16,8 @@
 
 trait BindTrait
 {
+    protected $id;
```

Comment:
the property is already defined in the classes that use the trait

Reply:
As above, the trait should define the properties it uses, no?